pkg/kms/api.go

```diff
@@ -127,10 +131,14 @@ const (
 	ECDSAP384TypeIEEEP1363 = KeyType(ECDSAP384IEEEP1363)
 	// ECDSAP521TypeIEEEP1363 key type value
 	ECDSAP521TypeIEEEP1363 = KeyType(ECDSAP521IEEEP1363)
+	// ECDSASecp256k1TypeIEEEP1363 key type value
+	ECDSASecp256k1TypeIEEEP1363 = KeyType(ECDSASecp256k1IEEEP1363)
```

Comment:
add a TODO here to support secp256K1 in kms/crypto's Tink implementation, I believe issue# is #1285